database/youtube/machine-learning.json

```
@@ -1,12 +1,19 @@
 [
-
-    {
-        "name":"Udacity",
-        "description":"Free online course for learning tensorflow for Deep learning",
-        "url":"https://learn.udacity.com/courses/ud187",
-        "category":"online platform",
-        "subcategory":"tensorflow",
-        "language":"english"
+    {
+        "name": "Two Minute Papers",
+        "description": "Artificial Intelligence and Machine Learning Research Papers explained Interestingly and in a Well-Detailed Manner. What a Time to be Alive!",
```

[Proofreading] Just some typos and unclear parts 
```suggestion
        "description": "These research papers discuss Artificial Intelligence and Machine Learning in an engaging and detailed manner. What a Time to be Alive!",
```

---

Just wanna suggest a small change here. I think it should be **This channel discusses Artificial Intelligence and Machine Learning Research Papers in an interesting and well-detailed manner**